src/test/java/io/airlift/units/TestDataSizeValidator.java

```diff
@@ -34,7 +34,12 @@
 
 public class TestDataSizeValidator
 {
-    private static final Validator VALIDATOR = Validation.byProvider(ApacheValidationProvider.class).configure().buildValidatorFactory().getValidator();
+    private static final Validator VALIDATOR = Validation.byProvider(HibernateValidator.class)
```

Comment:
Can you switch this to use `ValidationAssertions`?

Reply:
Done. PTAL @electrum